config_sample.json

```diff
@@ -0,0 +1,14 @@
+{
+  "peer_addrs": ["peer0.org1.example.com:7051","peer0.org2.example.com:9051"],
+  "orderer_addr": "orderer.example.com:7050",
+  "channel": "mychannel",
+  "chaincode": "mycc",
+  "version": "",
+  "args": ["query","a"],
+  "mspid": "Org1MSP",
+  "private_key": "./crypto-config/peerOrganizations/org1.example.com/users/User1@org1.example.com/msp/keystore/4d75bedcf454389483dfe0efbf12602b64d8c0e2903451c688b1d5d9ba38e655_sk",
```

Comment:
in newer fabric, private key file is renamed to `priv_sk` (which is quite convenient for us)